scss/styles.css

```diff
@@ -0,0 +1,686 @@
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .header__social {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-end;
+    align-items: center;
+    background-color: #312827;
+    color: #d3d3d3;
+    padding: 0.6em 0;
+  }
+
+  .header__social--ul {
+    width: 40%;
+    min-width: 350px;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .header__social--ul img {
+    width: 20px;
+    margin-left: 1em;
+    display: inline;
+  }
+
+  .header__social--ul .facebook {
+    width: 15px;
+  }
+}
+
+.headline {
+  padding: 100px 2em;
+  background-color: rgba(211, 211, 211, 0.8470588235);
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+@media screen and (min-width: 768px) {
+  .headline {
+    margin: 0;
+    padding: 5em 8em;
+  }
+}
+
+.headline h2 {
+  color: #ec5242;
+  font-weight: 400;
+  font-size: 1.3em;
+}
+
+@media screen and (min-width: 768px) {
+  .headline h2 {
+    color: #ec5242;
+    font-weight: 700;
+    font-size: 1.7em;
+  }
+}
+
+.headline h1 {
+  color: #ec5242;
+  font-weight: 900;
+  font-size: 3rem;
+}
+
+@media screen and (min-width: 768px) {
+  .headline h1 {
+    color: rgba(236, 83, 66, 0.6901960784);
+    font-weight: 900;
+    font-size: 4em;
+    background-image: url(../img/tekking.jpg);
+    background-size: contain;
+    background-repeat: repeat;
+    -webkit-background-clip: text;
+    background-clip: text;
+  }
+}
+
+.headline__p {
+  margin-top: 1em;
+  padding: 10px;
+  border: double 3px #fff;
+  box-shadow: 15px 15px 15px rgba(0, 0, 0, 0.15);
+  border-top-right-radius: 2rem;
+  border-bottom-left-radius: 2rem;
+  width: 90%;
+}
+
+@media screen and (min-width: 768px) {
+  .headline__p {
+    width: 50%;
+    background-color: rgba(255, 255, 255, 0.5803921569);
+  }
+}
+
+.headline h3 {
+  margin: 0.7em 0;
+  color: #312827;
+  font-weight: 900;
+  font-size: 1.8em;
+}
+
+.speakers {
+  padding-top: 20px;
+  background-color: rgba(211, 211, 211, 0.8470588235);
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.speakers > .speakersCreated {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 10px;
+}
+
+@media screen and (max-width: 768px) {
+  .speakers > .speakersCreated {
+    grid-template-columns: 1fr;
+    gap: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .speakers > .speakersCreated {
+    padding: 2em 0;
+  }
+}
+
+.speakers h2 {
+  color: #312827;
+  font-weight: 700;
+  font-size: 2rem;
+  margin-bottom: 1em;
+}
+
+.speakers img {
+  float: left;
+  margin: 10px;
+  display: inline;
+  width: 25%;
+  height: 8em;
+  position: relative;
+  -o-object-fit: fill;
+  object-fit: fill;
+  border-radius: 0 2rem 0 2rem;
+  border: solid 2px #312827;
+  box-shadow: 15px 15px 15px rgba(0, 0, 0, 0.15);
+}
```

Comment:
In the original design, there is a checquered background behind the image of the speaker which is missing in your design as shown below :point_down: . Kindly update accordingly. 

### Your design

![Screenshot from 2023-04-20 10-15-58](https://user-images.githubusercontent.com/65233860/233336312-8f033069-ff88-428f-9873-6e4b97e46d6a.png)


### Actual design 

![Screenshot from 2023-04-20 10-16-08](https://user-images.githubusercontent.com/65233860/233336345-58c34148-cd4c-4475-b365-98b43fc1147e.png)